lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
>  The behavior in Classic is that all four issues have the same fingerprint 

In general, if Classic behaves a certain way and we don't understand it well or have a definitive reason to believe it's buggy or inapplicable to Platform, we should err on the side of preserving it -- IMHO.

My memory is hazy, but I think the logic in the UI that collapses multiple instances of duplication within a single file to just one rendered issue (with other locations attached) relies on the fact that all the instances are the same fingerprint.

It also sounds like giving them all the same fingerprint means that, in your example @wfleming, it would not show up as a new issue since there's no new fingerprints in the *To* snapshot. That might actually make sense: it's not a new issue, you've just made an existing issue worse. I think that might even render as such since to remediation points would increase.